paddle/fluid/operators/row_conv_op.cu

```diff
@@ -31,8 +28,8 @@ template <typename T>
 __global__ void RowConvForwardSharedMemory(const T *in, const T *wt,
                                            int num_sequence, int input_dim,
                                            int future_context,
-                                           const size_t *batch_indices,
-                                           T *out) {
+                                           const size_t *batch_indices, T *out,
+                                           bool is_tensor, int timesteps) {
```

Comment:
There is no need to pass `boo is_tensor` and `int timesteps`. Since  `int current_timesteps = end - start;` is also adapte to padding.

Reply:
OK，fix it